asana/asana_test.go

```diff
@@ -14,6 +15,7 @@ var (
 	client *Client
 	mux    *http.ServeMux
 	server *httptest.Server
+	ctx    = context.Background()
```

Comment:
I know it's a test, but still, this isn't saving much typing. Instead of this variable here, just use `context.Background()` directly at the call sites. E.g.:

```Go
workspaces, err := client.ListWorkspaces(context.Background())
...
users, err := client.ListUsers(context.Background(), nil)
...
```